crates/libs/windows/src/core/generic_factory.rs

```diff
@@ -0,0 +1,28 @@
+use super::*;
+use std::mem::*;
+
+// A streamlined version of the IActivationFactory interface used by WinRT class factories used internally by the windows crate
```

Comment:
Nit: this should be a `///` doc comment so that it's picked up by tooling

Reply:
How does this work when the docs are hidden for this type?

Reply:
`rust-analyzer` shows hidden docs so those working on Windows (who would be the ones who would benefit from this doc), would be able to see this on hover.

Reply:
Ah that makes sense. I haven't used rust-analyzer in many months as it seemed to really struggle with the size of the repo.

Reply:
Anyway, I'll try to make a habit of using `///` for this purpose.

Reply:
Yes, I noticed rust-analyzer was really struggling... I was able to get it to work fine by editing the workspace definition and removing some things temporarily. This at least allowed me to use `rust-analyzer` from within `windows::core`.